Assets/Scripts/Blobs/BlobMagnet.cs

```diff
@@ -7,51 +7,36 @@ namespace Blobs
 {
     public class BlobMagnet : MonoBehaviour
     {
-        [Header("Sphere cast values")] [SerializeField]
-        private bool showGizmo;
-        [SerializeField] private float radius = 0.6f;
-
-
         [Header("Blob values")] [SerializeField]
         private float attractionSpeed;
 
         private Transform target;
         private Transform blobTransform;
 
-        //Check if it collides with the player, if it does
-
         private void Start()
         {
             blobTransform = transform;
```

Comment:
This seems unnecessary, why not just use transform across the script?

Reply:
Better for performance to not constantly retrieve the transform.